src/core/lib.lua

```diff
@@ -733,8 +733,6 @@ end
 function randomseed (seed)
    seed = tonumber(seed)
    if seed then
-      local msg = 'Using deterministic random numbers, SNABB_RANDOM_SEED=%d.\n'
```

Comment:
I think this is not a good change.  If a test spawns so many sub-processes with this environment variable set that it's too spammy, that test needs to filter sub-process output.